src/Account/Account.php

```diff
@@ -5,25 +5,18 @@
 /**
  * @author Milan Matějček
  */
-class Fio
+class Account
 {
+	/** @var Bank */
+	private $account;
 
 	/** @var string */
 	private $token;
 
-	/** @var Bank */
-	private $account;
-
-	public function __construct($token, Bank $account)
+	public function __construct($account, $token = NULL)
```

Comment:
I planned to merge this object with Bank, and thus it should be able to exist without token. Now I am thinking more about transforming Bank to some kind of Validator with static method, what do you think?